.claude/hooks/controller-annotation-check.sh

```
@@ -0,0 +1,69 @@
+if [[ "$FILE_PATH" != *Controller.kt ]]; then
+    exit 0
+fi
+
+# 파일이 존재하는지 확인
+if [[ ! -f "$FILE_PATH" ]]; then
+    exit 0
+fi
+
+echo ""
+echo "━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━"
+echo "📋 Controller Swagger 어노테이션 검증: $(basename $FILE_PATH)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)